### PR TITLE
Adding slncky recipe.

### PR DESCRIPTION
**Pull request by tstrnity**
* [ x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [x ] This PR adds a new recipe.
* [x ] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

File: recipes/slncky/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  fn: slncky-{{ version }}.tar.gz # [unix]
+  url: https://github.com/slncky/slncky/archive/{{ version }}.tar.gz # [unix]
+  md5: b8a1b22acd355851d18a40fb06e7f510 # [unix]
+
+build:
+    number: 0
+
+requirements:
+    build:
+        - python >=2.7,<3
```
Comment (bgruening) [inline recipes/slncky/meta.yaml]: please skip the build in the build section for py3k and remove the pinning here.

File: recipes/slncky/post-link.sh [hunk 2]
```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+echo "It is necessary to upload annotations. This can take awhile and when expanded uses about 12GB."
```
Comment (bgruening) [inline recipes/slncky/post-link.sh]: s/upload/download/ ?

File: recipes/slncky/run_test.sh [hunk 3]
```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+set -x
+set -e
+
+echo "PATH is $PATH"
+# The annotations directory needs to be uploaded by the user.
+# It is too big to be uploaded automatically.
+# test -d $PREFIX/annotations
+slncky --help > /dev/null 2>&1
```
Comment (bgruening) [inline recipes/slncky/run_test.sh]: can you this simply put in the meta.yaml file in a test section?

File: recipes/slncky/post-link.sh (hunk 2 above)
Comment (tstrnity) [reply recipes/slncky/post-link.sh]: yes.

File: recipes/slncky/run_test.sh (hunk 3 above)
Comment (tstrnity) [reply recipes/slncky/run_test.sh]: Yes. I will do that.

File: recipes/slncky/meta.yaml (hunk 1 above)
Comment (tstrnity) [reply recipes/slncky/meta.yaml]: ok.

File: recipes/slncky/run_test.sh (hunk 3 above)
Comment (tstrnity) [inline recipes/slncky/run_test.sh]: I pushed my changes into the slncky branch. Not sure if I need to close this pull request and open another one, or if the changes are now included with this pull request.

Comment (tstrnity) [reply recipes/slncky/run_test.sh]: Seems like they are incorporated into this pull request.

**Comment by bgruening** [comment]
Thanks @tstrnity!

**Comment by tstrnity** [comment]
Thanks for the change on the python. I wasn't sure exactly what was needed, and your change clarified that for me.

**Pull request merged**